pkg/docker/hub/tests/init/hello_world.sql

```diff
@@ -0,0 +1 @@
+select version();
```

Comment:
I don't see `hello_world.sql` being used anywhere. I'm assuming it's there for convenience to manually run against a running container, but if it's not referenced anywhere let's leave it out.